packages/dev/core/src/Engines/nullEngine.ts

```diff
@@ -152,6 +152,7 @@ export class NullEngine extends Engine {
             supportSRGBBuffers: false,
             supportTransformFeedbacks: false,
             textureMaxLevel: false,
+            texture2DArrayMaxLayerCount: this._webGLVersion > 1 ? 256 : 128,

```

Comment:
not sure what it means here to have a webglversion (for nullengine we do't care)